serialization/dotnet/json/src/JsonSerializationWriter.cs

```diff
@@ -246,12 +246,6 @@ private void WriteNonParsableObjectValue<T>(string key, T value)
         }
         private void WriteAnyValue<T>(string key, T value)
         {
-            if(value == null)
```

Comment:
Removed as there is already a null case in the switch case below. This caused null properties to be repeated in the final payload.

Reply:
thanks! going too fast on my side....